data/remnant ships.txt

```diff
@@ -29,7 +29,7 @@ ship "Albatross"
 		"heat dissipation" 0.6
 		"fuel capacity" 600
 		"ramscoop" 2
-		"outfit space" 658
+		"outfit space" 668
```

Comment:
It shouldn't be an issue to install an outfits expansion where necessary instead of buffing the base stats, except for perhaps the heavy variant, which I'd say could go without a scram drive.

Reply:
The ship already something of a hot cannon, in stock configuration it launches with over half heat and shuts down due to overheat within a couple minutes of intense use. I'd say that yes, an outfit expansion is an issue for it. Even after this change, it still has 16% less space than the similarly price Shield Beetle. (that'd be 130 tons of space)

I'm not really arguing for a major buff to the Albatross, just pointing out that it is more expensive than the SB and also license locked, and has 130 tons less space. Sure, it has built in stuff, but that's not much of an advantage compared to "game-provided means of easily acquiring dozens or hundreds of SBs for free anytime you want." Not to mention that the SB has higher engine, weapon, and cargo space, too. Given how the two stack up, I don't think that 10 tons for a scramdrive is exactly throwing the balance between the two of these off. 

That being said, adjusted the price up slightly on both ships.